### PR TITLE
Upgrades

### PR DESCRIPTION
**Pull request by praseodym**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -30,23 +30,24 @@ repositories {
...
 configurations {
-    // We're using Jetty instead
-    compile.exclude module: "spring-boot-starter-tomcat"
     // We're using HikariCP instead
     compile.exclude module: "tomcat-jdbc"
 }
 
 dependencies {
     compile("org.springframework.boot:spring-boot-starter-data-jpa")
-    compile("org.springframework.boot:spring-boot-starter-jetty")
```
Comment (RickWieman) [inline build.gradle]: We switch back to tomcat? Is there any reason for that? (Just curious)

File: src/test/java/ch/wisv/events/EventsApplicationTests.java [hunk 2]
```diff
@@ -29,29 +28,28 @@
 import static org.hamcrest.core.IsCollectionContaining.hasItem;
 
 @RunWith(SpringJUnit4ClassRunner.class)
-@SpringApplicationConfiguration(classes = EventsApplication.class)
-@WebAppConfiguration
+@SpringBootTest(classes = EventsApplication.class, webEnvironment = SpringBootTest.WebEnvironment.NONE)
```
Comment (RickWieman) [inline src/test/java/ch/wisv/events/EventsApplicationTests.java]: Cool!

File: build.gradle (hunk 1 above)
Comment (praseodym) [inline build.gradle]: Not really, but Tomcat has `javax.validation` bundled which was somehow missing after upgrading to Spring Boot 1.4.0.RELEASE with Jetty.

**Comment by RickWieman** [comment]
:shipit:

**Pull request merged**